### ISSUE TITLE
update database

### ISSUE THREAD

**Issue by urb1s** [Jimk72/Icarus_Software#12]
Hey, i downloaded your mod manager and im doing everything step by step like you in the video but i got an issue with updating a database i cant really find anyone with this issue so i dont know how to really solve it, it always says 

"Trying to Download: https://firestore.googleapis.com/v1/projects/projectdaedalus-fb09f/databases/(default)/documents/mods?pageSize=300
Can not download: https://firestore.googleapis.com/v1/projects/projectdaedalus-fb09f/databases/(default)/documents/mods?pageSize=300
No mods found in memory!" can you help me out with this thing to solve it please?

**Comment by Jimk72** [comment]
Are you running as admin and is there any restrictions for internet access?

**Comment by urb1s** [comment]
Yeah i am running it as admin as well but it doesnt solve the issue, the same goes for internet access restrictions i dont think i have any, only thing i have turned on is windows defender.

**Comment by Jimk72** [comment]
When you click that link does it bring up the json file with all the mods?

**Comment by urb1s** [comment]
I mean only thing i can do with that link is i can either copy it and paste it in browser to see some kind of links and things but i cant really do something with it im just clueless.

**Comment by Jimk72** [comment]
![image](https://github.com/user-attachments/assets/78537e0e-f0ce-4910-90ff-794497697c28)

**Comment by Jimk72** [comment]
This is what I've found online. The code signing costs too much money for me.

**Comment by Jimk72** [comment]
My latest ver has a new downloader so It may fix the issues you had.
